### DISCUSSION TITLE
Question about Mastodon Glitch Edition

### DISCUSSION THREAD

**Discussion by NSFWHubSocial** [mastodon/mastodon#21515] [Q&A] [answered]
Does the Glitch Edition support larger media files? Also, is embedded HTML supported in posts? This would be awesome! Lastly, is there anyone available to install the Glitch Edition for me and how much would the task cost?

**Comment by Lastorder-DC** [comment]
https://github.com/glitch-soc/mastodon/ Ask glitch-soc's repo. not here.

**Comment by nightpool** [comment, marked as answer]
Glitch is a fork of Mastodon, you should ask them about what they support, there's no way for us to answer that. (although there is some overlap between the maintainers because of attrition)
